internal/util/strings.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2021 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+// ConstainsInSlice find given string in given slice
+func ContainsInSlice(slice []string, s string) bool {
+	for _, item := range slice {
+		if item == s {
+			return true
+		}
+	}
+
+	return false
+}
+
+// RemoveFromSlice removes given string from given slice
+func RemoveFromSlice(slice []string, s string) (result []string) {
```

Comment:
please do not use these implicit return variables, we don't use it anywhere else either. declare `result` and return it.